### DISCUSSION TITLE
Highlight code output

### DISCUSSION THREAD

**Discussion by Cghlewis** [quarto-dev/quarto-cli#2387] [Q&A] [answered]
Hi!
I am wondering if there is a way to highlight code output (not the code itself-just the output) in a Quarto Revealjs presentation? I used to add the code chunk option `highlight.output` in R Markdown but that doesn't seem to work now? Thank you!

![rmarkdown_option](https://user-images.githubusercontent.com/33465957/189422617-103bb46a-79f7-4fc8-9e7a-0e514c17ee19.PNG)
![output_highlight](https://user-images.githubusercontent.com/33465957/189422633-54318f1b-b214-406d-9c50-6c7df95145ee.PNG)

**Comment by mcanouil** [comment]
From the documentation, there is currently no way to do that.

Using css with the `class-*` and `attr-*` can change the whole output but not particular lines.

I think this could be a feature request to extend `code-lines-number` to "cell output" (<https://quarto.org/docs/reference/cells/cells-knitr.html#cell-output>).

**Reply by Cghlewis** [reply]
Thank you!

**Reply by mcanouil** [reply, marked as answer]
I opened a feature request issue: #2399 (you can see it on the right below the menu in the "desktop" version of GitHub)

**Reply by Cghlewis** [reply]
Thank you! I wasn't 100% certain how to do it, so I appreciate it.
